src/config/config-factory.ts

```diff
@@ -4,11 +4,43 @@ import kenyaConfig from './chis-ke';
 import togoConfig from './chis-tg';
 import civConfig from './chis-civ';
 
+export enum Feature {
+  Create = 'create',
+  ReplaceContact = 'replace-contact',
+  Move = 'move',
+}
+
+const parseConfig = (c: any): PartnerConfig => {
+  return {
+    config: {
+      ...c.config,
+      contact_types: c.config.contact_types.map((t: any) => {
+        return {
+          ...t,
+          feature_flags: t.feature_flags?.map((v: string) => {
+            if ((Object.values(Feature) as string[]).indexOf(v) === -1) {
+              throw new Error(
+                'invalid feature flag: ' +
```

Comment:
Can we use a config syntax which doesn't need parsing? What about three booleans: `can_edit`, `can_replace` and `can_move`?

Reply:
Yeah that's more readable, wanted to have a shorter config file but might not be worth it